core/src/test/java/io/snappydata/app/JavaCreateIndexTestSuite.java

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
I think the overall formatting of this file is not proper. Please close Idea, copy "codeStyleSettings.xml" inside .idea directory in the checkout, then restart. Hitting a Ctrl-Shift-F will format the entire file (when using eclipse key bindings) or as per key binding you have set.